index.src.html

```diff
@@ -2090,6 +2092,22 @@
     encouraged to check by calling {{AudioEncoder/isConfigSupported()}} with
     |config|.
   </dd>
+  <dt><dfn dict-member for=AudioEncoderConfig>contentHint</dfn></dt>
+  <dd>
+    An encoding [=audio content hint=] as defined by [[mst-content-hint]].
+
```

Comment:
Is it confusing that a User Agent can ignore the hint (MUST NOT refuse the config if it's unsupported) even if there is a registry entry which MUST be considered?

Reply:
yeah, it is kind of confusing. see my comment below.

Reply:
See other comment.